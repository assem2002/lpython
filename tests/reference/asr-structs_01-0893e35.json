{
    "basename": "asr-structs_01-0893e35",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/structs_01.py",
    "infile_hash": "9342dd6345fb71bbbfb1c41e643530c58493758d08774f838b1c85e0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-structs_01-0893e35.stderr",
    "stderr_hash": "a9e672c827ab451b501313e0b9cdd756a2be77bb1d074c27c4f13e10",
    "returncode": 2
}