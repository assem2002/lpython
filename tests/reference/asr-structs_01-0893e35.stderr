semantic error: Alignment 5 is not a positive power of 2.
 --> tests/errors/structs_01.py:3:17
  |
3 | @packed(aligned=5)
  |                 ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
